.travis.yml

```diff
@@ -7,19 +7,13 @@ addons:
   chrome: stable
 
 cache:
-  bundler: true
```

Comment:
We don't use Ruby so we don't need to cache Bundler. Presumably this is a holdover from when Compass was all the rage.